client/packages/host/src/components/Initialise/hooks.ts

```diff
@@ -98,10 +99,12 @@ export const useInitialiseForm = () => {
   const { data: initStatus } = useInitialisationStatus(refetchInterval);
   const { data: syncStatus } = useHost.utils.syncStatus(refetchInterval);
   const { data: syncSettings } = useHost.settings.syncSettings();
+  const { acquireWakeLock, releaseWakeLock } = useNativeClient();
```

Comment:
I think if we are doing this for just initialisation screen and manual sync, we can use: https://github.com/capacitor-community/keep-awake.

Reply:
I think acquireWakeLock would be useful if we want to sync `all the time` when app is in the background

Reply:
yeah, keep-awake should also work. That would require adding another plugin (package maintenance is a headache) and would drain the battery faster than a wakelock, as it will keep the screen on. 

The advantage is that it would be a cross-device solution; would work for IOS as well as android. which we aren't planning to support any time soon.

Reply:
> as it will keep the screen on

I thought it's fine for initialisation and manual sync, or do we expect this to be an issue for the user ?

> package maintenance is a headache

That kind of implies that we should make our own plugins and packages in he future rather then lookup up existing ones ?

> The advantage is that it would be a cross-device solution

I think the main advantage would be the time implementing and time and review time (can also say disadvantage is we would have to trust package maintainer). 

Happy with implementing our own plugins, as long as we have a general guidelines/understanding of when to implement our own vs when to use existing (would assume in this case it's good choice to implement, since it's not much code and alternative is not official capacitor one)